vendor.conf

```diff
@@ -44,7 +44,7 @@ google.golang.org/appengine 12d5545dc1cfa6047a286d5e853841b6471f4c19
 google.golang.org/cloud 975617b05ea8a58727e6c1a06b6161ff4185a9f2
 google.golang.org/grpc d3ddb4469d5a1b949fc7a7da7c1d6a0d1b6de994
 gopkg.in/check.v1 64131543e7896d5bcc6bd5a76287eb75ea96c673
-gopkg.in/square/go-jose.v1 40d457b439244b546f023d056628e5184136899b
+gopkg.in/square/go-jose.v1 aa2e30fdd1fe9dd3394119af66451ae790d50e0d # v1.1.0
```

Comment:
Looks like this is a dependency for for rsc.io/letsencrypt, which has been removed on master (through https://github.com/docker/distribution/pull/2926), and is being discussed to remove for the 2.7 release as well (see https://github.com/docker/distribution/issues/1976)

Perhaps instead we should backport https://github.com/docker/distribution/pull/2926 to this branch ?

@dmcgowan ptal

Reply:
With square/go-jose.v1 removed on master, what would be the preferred path forward on this? 

- Create a PR only for the satori/go.uuid bump on master +  backport to release/2.7.
- The square/go-jose.v1 change would be only for release/2.7.

Reply:
I opened https://github.com/docker/distribution/pull/3134 for consideration by the maintainers

For the satori/go.uuid package; yes, having a PR for master, and a "backport" to the 2.7 branch would be the preferred approach. Unfortunately, a straight backport won't be possible, given that "master" switched to using go modules (but the 2.7 branch not yet). Go modules likely make it slightly less "clean", given that satori/go.uuid is not really maintained, and hasn't produced a tag with the fix (ideally, someone would request them to tag a new release); for go mod, updating likely involves some override to be added to go.mod.

We should also make sure that the upstream Azure repository is updated to use a version with the fix; https://github.com/Azure/azure-sdk-for-go/blob/df064cee2cbebe4f78158351469d9afbad020c06/Gopkg.lock#L112-L118

Reply:
Thanks, #3135 created for master.

Getting rid of square/go-jose on 2.7 works for me.  I'll refine this PR to just address the satori/go.uuid change.